cypress/PageObjects/deleteProductsPage.js

```diff
@@ -0,0 +1,19 @@
+///reference types="Cypress"
+
+class DeleteProductsPage {
+
+    get deleteFirstElement(){
+        return cy.get('[aria-label="Remove Pixel 6a 128GB Chalk (Unlocked) from cart"]')
```

Comment:
implement a method that removes product by it's name.
Remove Pixel 6a 128GB Chalk (Unlocked) from cart"] is a parameter value

it is not a locator

Reply:
deleteProductFromCard(productName)